### PR TITLE
Minor clarification for why sudo is needed

### PR DESCRIPTION
**Pull request by bbarker**
Also, adding some nix build instructions.

**Description of the Pull Request (PR):**

Write your description of the PR here. Be sure to include as much background,
and details necessary for the reviewers to understand exactly what this is
fixing or enhancing.

**This fixes or addresses the following GitHub issues:**

- Ref: #

**Checkoff for all PRs:**

- [x] I have read the [Guidelines for Contributing](https://github.com/singularityware/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- [ ] I have added changes to the [CHANGELOG](https://github.com/singularityware/singularity/blob/development/CHANGELOG.md) and and documentation updates to the [singularityware](https://www.github.com/singularityware/singularityware.github.io) documentation base.
- [x] I have tested this PR locally with a `make test`
- [ ] This PR is NOT against the project's master branch
- [ ] I have added myself as a contributor to the [contributors's file](https://github.com/singularityware/singularity/blob/master/CONTRIBUTORS.md)
- [ ] This PR is ready for review and/or merge

Attn: @singularityware-admin

### CODE REVIEW

**Comment by bbarker** [comment]
Since this is a minor change (doc changes), not sure if contributor's file update is necessary.

**Comment by bbarker** [comment]
Only issue in testing so far with nixpkgs instructions, when I run `make test`, squashfs-tools isn't available currently, so I will see about fixing this "upstream" (in nixpkgs).

**Comment by bbarker** [comment]
TODO: plan to rebase against development-2.x on local checkout

File: INSTALL.md [hunk 1]
```diff
@@ -39,7 +39,31 @@ $ sudo make install
 </pre>
 
 note: The `sudo` is very important for the `make install`. Failure to do this
-will result in a non-functioning or semi-functioning installation.
+will result in a non-functioning or semi-functioning installation. This is required
+due to Singularity requiring the `SUID` bit to be set, even if the user installs
+into a directory in which she has full priviledges.
+
+
+### Using nixpkgs to build
```
Comment (jmstover) [inline INSTALL.md:47]: Follow up from Slack chat...

* Rebase against `development-2.x`
* Remove the `Using nixpkgs to build` section

Hrrmm...  As for linking examples.... I suppose that if we crate a new file `PKG_EXAMPLES.md`  (or similar), and within that have some known examples listed like:

```
## Packaging
For example Singularity packaging see:
Example [RPM Spec File](https://github.com/singularityware/singularity/blob/master/singularity.spec.in)
Example [Debian Package Structure](https://github.com/singularityware/singularity/blob/master/debian)
Example [Alpine examples](https://github.com/singularityware/singularity/blob/master/alpinelinux)

### Tools
[easybuild](https://github.com/easybuilders/easybuild-easyconfigs/tree/master/easybuild/easyconfigs/s/Singularity)
[nixpkgs](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/virtualization/singularity/default.nix)
```

That would probably be fine. But we were mostly trying to keep the INSTALL.md minimal.

**Review by jmstover — CHANGES_REQUESTED** [review]

**Pull request closed without merging**